internal/saucecloud/cloud.go

```diff
@@ -507,6 +469,53 @@ func (r *CloudRunner) remoteArchiveFiles(project interface{}, files []string, sa
 	return strings.Join(uris, ","), nil
 }
 
+// GetJUnitArtifacts retrieves the JUnit report and parent JUnit reports for the current job.
+func (r *CloudRunner) GetJUnitArtifacts(res result) (junits []report.Artifact, parentJUnits []report.Artifact) {
```

Comment:
Why is `junits []report.Artifact` a slice? I only ever see one value added.

Reply:
Will use `report.Artifact` instead.